src/handlers/bridge.lua

```diff
@@ -1,5 +1,9 @@
 
 -- Patch require to log all executed modules
+_G._require = require
```

Comment:
Any reason to have this while its on line 7 as well?

Reply:
None, my mistake